Utils/createBenchmark.py

```diff
@@ -0,0 +1,61 @@
+# python3 createBenchmark.py MyRegexBenchmark
```

Comment:
If the benchmarks just added themselves to an array, would that simplify the registration process?